app/model/editions/templates/DailyEdition.scala

```diff
@@ -618,16 +652,16 @@ object FrontSportObserver {
   )
 }
 
-object FrontSpecialSpecial2 {
-  val collectionSpecialSpecial2 = CollectionTemplate(
+object FrontSpecialSpecial4 {
+  val collectionSpecialSpecial4 = CollectionTemplate(
     name = "Special",
     prefill = Some(CapiPrefillQuery("?tag=theguardian/special-supplement/special-supplement|theobserver/special-supplement/special-supplement", PathType.PrintSent)),
     presentation = TemplateDefaults.defaultCollectionPresentation
   )
 
   val front = FrontTemplate(
-    name = "Special 2",
-    collections = List(collectionSpecialSpecial2),
+    name = "Special 4",
+    collections = List(collectionSpecialSpecial4),
     presentation = TemplateDefaults.defaultFrontPresentation,
```

Comment:
Ditto - should this be the default formatting?

Reply:
I think it's okay - default formatting gives the neutral palette (I think!) - it's only the two specials that straddle Top stores that want the new "special" palette. Perhaps I should name them differently?